File: TFC-TourGuide-Android/app/src/main/java/com/example/tourguide/modelo/Propuesta.java
package com.example.tourguide.modelo;

public class Propuesta {
    private String id, idGuia,idAnuncio,mensaje;
    private Estados estado;
    private Guia guia;

    public Propuesta(String id, String idGuia, String idAnuncio, String mensaje,Estados estado,Guia guia) {
        this.id = id;
        this.idGuia = idGuia;
        this.idAnuncio = idAnuncio;
        this.mensaje = mensaje;
        this.estado = estado;
        this.guia = guia;
    }

    public Estados getEstado() {
        return estado;
    }

    public void setEstado(Estados estado) {
        this.estado = estado;
    }

    public Guia getGuia() {
        return guia;
    }

    public void setGuia(Guia guia) {
        this.guia = guia;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getIdGuia() {
        return idGuia;
    }

    public void setIdGuia(String idGuia) {
        this.idGuia = idGuia;
    }

    public String getIdAnuncio() {
        return idAnuncio;
    }

    public void setIdAnuncio(String idAnuncio) {
        this.idAnuncio = idAnuncio;
    }

    public String getMensaje() {
        return mensaje;
    }

    public void setMensaje(String mensaje) {
        this.mensaje = mensaje;
    }

    @Override
    public String toString() {
        return "Propuesta{" +
                "id='" + id + '\'' +
                ", idGuia='" + idGuia + '\'' +
                ", idAnuncio='" + idAnuncio + '\'' +
                ", mensaje='" + mensaje + '\'' +
                ", estado=" + estado +
                ", guia=" + guia +
                '}';
    }
}

File: TFC-TourGuide-PHP/modelos/Propuesta.php
<?php

// Esta clase representa una Usuario

class Propuesta
{
    // Variables de clase
    private $idguia,$idanuncio ,$mensaje,$estado;

    // Constructor
    public function __construct($idguia,$idanuncio, $mensaje,$estado)
    {
        $this->idguia = $idguia;
        $this->idanuncio = $idanuncio;
        $this->mensaje = $mensaje;
        $this->estado = $estado;
    }

    public function getEstado(){
        return $this->estado;
    }

    public function getIdGuia(){
        return $this->idguia;
    }

    public function getIDAnuncio(){
        return $this->idanuncio;
    }

    public function getMensaje(){
        return $this->mensaje;
    }
    
    
    // Muestra los datos de la Usuario
    public function toString()
    {
        return
            [
                "idguia" => utf8_encode($this->idguia),
                "idanuncio" => utf8_encode($this->idanuncio),
                "mensaje" => utf8_encode($this->mensaje),
                "estado" => utf8_encode($this->estado)
            ];
    }
}